src/libraries/Microsoft.Extensions.Logging.Console/src/Microsoft.Extensions.Logging.Console.csproj

```diff
@@ -58,6 +52,15 @@
     <ProjectReference Include="$(LibrariesProjectRoot)System.Text.Json\src\System.Text.Json.csproj" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp' and '$(TargetFrameworkVersion)' == 'v6.0'">
```

Comment:
Shouldn't this be

`!$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net7.0'))`

?

Reply:
Both the .NETStandard and the .NETFramework tfms don't depend on Unsafe.